text-ui-test/input.txt

```diff
@@ -0,0 +1,49 @@
+todo
+todo 
+todo dfsfsdf
+event
+event 
```

Comment:
It is good that you have so many test cases, this shows that you test your codes thoroughly :)